lib/libutils/isoc/include/assert.h

```diff
@@ -12,17 +12,20 @@ void __noreturn _assert_break(void);
 void _assert_log(const char *expr, const char *file, const int line,
 			const char *func);
 
+static inline void __noreturn _assert_trap(const char *expr_str,
+					   const char *file, const int line,
+					   const char *func)
+{
+	_assert_log(expr_str, file, line, func);
+	_assert_break();
+}
+
 /* assert() specs: generates a log but does not panic if NDEBUG is defined */
 #ifdef NDEBUG
-#define assert(expr)	do { } while (0)
+#define assert(expr)	((void)0)
```

Comment:
The comment is wrong, no log is ever generated when `NDEBUG` is defined. Please just remove the comment.

Reply:
Ok, agree.